src/redux/explorer.js

```diff
@@ -246,7 +247,12 @@ const isValidAssetsResponseFromZerion = msg => {
   return false;
 };
 
-export const explorerClearState = () => dispatch => {
+export const explorerClearState = () => (dispatch, getState) => {
+  const { fallback } = getState().explorer;
+  if (fallback) {
+    logger.log('😬 Disabling fallback data provider!');
+    dispatch(fallbackExplorerClearState());
```

Comment:
`fallbackExplorerClearState` is already done as part of `disableFallbackIfNeeded` in line 256

Reply:
oops